permissions/plugin-aws-sqs.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "aws-sqs-plugin"
+paths:
+- "org/jenkins-ci/plugins/aws-sqs-plugin"
```

Comment:
Sorry to be pedantic, but I'm not sure what you're asking. Are you saying:

**`artifactId` should match `name`?**

i.e. change
```Xml
<artifactId>aws-sqs</artifactId>
```
to
```Xml
<artifactId>aws-sqs-plugin</artifactId>
```

**`groupId` should match `paths`?**

i.e. change
```Xml
<project<groupId>io.relution.jenkins</groupId>
```
to
```Xml
<project><groupId>org.jenkins-ci.plugins</groupId>
```

Or vice versa (change the yml to match the pom)?